src/components/PackageCard/PackageCard.js

```diff
@@ -39,6 +39,17 @@ const PackageCard = ({
 }) => {
   const pkgObject = pkg._object ?? pkg;
 
+  let synchronisationStatus;
+  if (pkgObject?.syncContentsFromSource !== null) {
+    if (pkgObject?.syncContentsFromSource) {
+      synchronisationStatus = <FormattedMessage id="ui-agreements.eresources.synchronising" />;
+    } else {
+      synchronisationStatus = <FormattedMessage id="ui-agreements.eresources.paused" />;
+    }
+  } else {
+    synchronisationStatus = <NoValue />;
```

Comment:
See above... I don't want us to repeat this kind of logic, when we make changes these things break. Utility component please `PackageSyncronisingValue` or something